canonical/ubuntu.com
# Implement /credentials/shop/ as flask view

## Done

- Removed previous `/credentials/shop/` that was previously a React app
- Implemented in flask view

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Go to `/credentials/shop/`
- Purchase an exam
- Should redirect to the checkout page with the correct exam & price

## Issue / Card

Fixes [WD-8193](https://warthogs.atlassian.net/browse/WD-8193)

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
